Kernel/System/PostMaster/NewTicket.pm

```diff
@@ -457,12 +457,22 @@ Message
                 ID => $DynamicFieldID,
             );
 
-            $DynamicFieldBackendObject->ValueSet(
-                DynamicFieldConfig => $DynamicFieldGet,
-                ObjectID           => $TicketID,
-                Value              => $GetParam{$Key},
-                UserID             => $Param{InmailUserID},
-            );
+            if ($DynamicFieldGet->{FieldType} eq "Multiselect") {
```

Comment:
```suggestion
            # If the dynamic field is a multiselect, we need to split the value into an array
            if ($DynamicFieldGet->{FieldType} =~ /Multiselect$/) {
```

You can find all available DynamicField drivers here:
Kernel/System/DynamicField/Driver/*

- Multiselect.pm
- WebserviceMultiselect.pm

There may of course be other drivers due to installed packages.